src/components/DataGridWithInfiniteScroll/DataGridWithInfiniteScroll.tsx

```diff
@@ -121,7 +256,43 @@ const DataGridWithInfiniteScroll = (props: DataGridProps) => {
     }
 
     return (
-        <div className="table-css">
+
+    <div className='clr-row flex-container'>
+        {isOpen && 
+            <div
+            className="column-switch clr-popover-content"
+            ref={refChild}
+            role="dialog"
+            style={{
+                top: 0,
+                bottom: "auto",
+                right: "auto",
+                left: 0,
+                transform: transformVal,
+            }}>
+              <div className="ColumnSelect" >
+                <span className="ColumnSelect-header">Column Picker</span>
+
+                {allColumns.map((column) =>
+                    <div key={column.id}>
+                        <label>
+                            <input type='checkbox' id="allColumnSelect"  {...column.getToggleHiddenProps()} />
+                            <span className="ColumnSelect-header-names">{column.Header}</span>
+                        </label>
+                    </div>
+                )}
+          </div>
+                <div className="ColumnSelect-reset-button">
+            
+                    <Button link className="Reset-button-margin-css"  >
+                    <span className="ColumnSelect-header">Reset Settings </span> </Button>
+
+                </div>
+          </div>
+        }
+
+        <div className="table-css" style={props.style} ref={refParent}>
+            <div className={showDetailsPanel ? 'clr-col-12' : 'clr-col-12'}>
```

Comment:
What is use of `?` operator here? we are using same CSS Class for both condition